code/index.html

```diff
@@ -1,22 +1,168 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <link rel="stylesheet" href="Style.css">
+      <link href="https://fonts.googleapis.com/css2?family=Cormorant+Garamond:wght@600&display=swap" rel="stylesheet">
+      <title>Stockholm Times</title>
+    </head>
+    
+    <body>
+      <div class="pagewraper">
+    
+        <!--HEADER SECTION-->
+        <header class="header">
+          <div class="header__top">
```

Comment:
A better practise when naming classes is you use "kebab letters" which is when you use the - between words, here for example you could name it header-top. It's just a minor detail but it's good to know 👩‍💻